recipes/wdltools/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "wdlTools" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dnanexus-rnd/wdlTools/archive/{{ version }}.tar.gz
+  sha256: 115fd4ce8ae76e2963b284617bc14c72f494b11a4bef51c67baabd95d2a01974
+
+build:
+  skip: true  # [not unix]
+  number: 0
+
+requirements:
+  build:
+    - openjdk =8
+    - sbt
+    - make
+    - wget
+  run:
+    - openjdk =8|11
```

Comment:
Is this a valid spec?  would `!9,!10` work?

Reply:
Yes it is.
8 and 11 are LTS releases, so they are explicitly asked for here.